infra/resources/prod/redis.tf

```
@@ -46,49 +46,49 @@ module "redis_messages" {
 }
 
 
-module "com_redis_messages" {
-  source = "github.com/pagopa/terraform-azurerm-v4//redis_cache?ref=v1.2.1"
+resource "azurerm_redis_cache" "com_redis_messages" {
```

```suggestion
resource "azurerm_redis_cache" "com" {
```

`_redis_message` is redundant here